hack/aotutil/.gitignore

```diff
@@ -0,0 +1,5 @@
+# binary
+aotutil
+
```

Comment:
can we not use `hack` as folder name?

Reply:
There are other projects using hack folder as well, e.g. k8s https://github.com/kubernetes/kubernetes/tree/master/hack

Though we can change it to something like `tools` or move it to a `cmd/aotutil` so it's more go project style, but this is a java project ...

Reply:
Moved to cmd cc @bjrara PTAL